pandas/core/generic.py

```diff
@@ -3296,12 +3296,16 @@ def fillna(self, value=None, method=None, axis=None, inplace=False,
             return self._constructor(new_data).__finalize__(self)
 
     def ffill(self, axis=None, inplace=False, limit=None, downcast=None):
-        """Synonym for NDFrame.fillna(method='ffill')"""
+        """
+        Synonym for :meth:`DataFrame.fillna(method='ffill') <DataFrame.fillna>`
```

Comment:
can you add a ``Paramaters/Returns`` section here (you can use the shared docs fillna as an Appender), though may need a modification